src/services/LiquidityService/LiquidityService.js

```diff
@@ -10,6 +10,8 @@ const assert = require('assert')
 
 const MAXIMUM_DX_FEE = 0.005 // 0.5%
 const WAIT_TO_RELEASE_SELL_LOCK_MILLISECONDS = process.env.WAIT_TO_RELEASE_SELL_LOCK_MILLISECONDS || (2 * 60 * 1000) // 2 min
+// Release time if a sell is locked (never commiting because a reorg or transaction loss)
+const EMERGENCY_RELEASE_SELL_LOCK_MILLISECONDS = process.env.EMERGENCY_RELEASE_SELL_LOCK_MILLISECONDS || (15 * 60 * 1000) // 15 min
```

Comment:
Is this being implemented for only sell liquidity? I think most of bots share this logic. If not for this PR at some point we should extract all the coordination/lock/etc in utilities that all bots can use. For example buy bot, sell bot, arbitrage bot, etc don't need to replicate this logics

Reply:
Also, I find the EMERGENCY word a bit too strong :) 
I think it's more like a maximun wait time for the operation.